shotover-proxy/src/transforms/redis/sink_single.rs

```diff
@@ -35,20 +39,27 @@ pub struct RedisSinkSingle {
     address: String,
     tls: Option<TlsConnector>,
     outbound: Option<Framed<Pin<Box<dyn AsyncStream + Send + Sync>>, RedisCodec>>,
+    chain_name: String,
 }
 
 impl Clone for RedisSinkSingle {
     fn clone(&self) -> Self {
-        RedisSinkSingle::new(self.address.clone(), self.tls.clone())
+        RedisSinkSingle::new(
+            self.address.clone(),
+            self.tls.clone(),
+            self.chain_name.clone(),
+        )
     }
 }
 
 impl RedisSinkSingle {
-    pub fn new(address: String, tls: Option<TlsConnector>) -> RedisSinkSingle {
+    pub fn new(address: String, tls: Option<TlsConnector>, chain_name: String) -> RedisSinkSingle {
+        register_counter!("redis_single_failed_request", Unit::Count, "chain" => chain_name.clone());
```

Comment:
Naming wise we should call this "failed_requests" and add a tag "transform" => "redis_sink_single"

Reply:
Should redis cluster also have these changes?